Added new Types: ShortStory, Manuscript, Poster, Drawing, Play, SheetMusic to bib.schema.org

RichardWallis opened the pull request:
Replacement for PR #1449  to be in step with master after 3.4 release.
Adds new subtypes of CreativeWork: ShortStory, Manuscript, Poster, Drawing, Play, SheetMusic

Implements issues #1448 & #1976

danbri commented:
LGTM. How should we handle updates to docs/releases.html ... we often end up with changes merged in that don't have corresponding releases text. But I think we've had conflicts when several parallel PRs all try to propose edits to the releases page. Maybe drafting the releases text here is a reasonable midway? /cc @vholland

Dataliberate commented:
Addressing the [vocabulary editor’s] challenge of capturing all the changes
into docs/releases.html would be good.

You are right that getting individual edits to the document to merge
together from several parallel PRs is equally challenging, especially as we
tend to save up PRs to batch apply just ahead of a new version.

I did take a look at this:
https://github.com/github-tools/github-release-notes

But like anything of this nature, there needs to be some discipline and
consistency from those committing to the repo.

Drafting a releases text into the comments associated with a commit and/or
PR is probably a good compromise.  The person responsible for PR merging
could use the lack of such text as a reason for not committing.

~Richard.

Richard Wallis
Founder, Data Liberate
http://dataliberate.com
Linkedin: http://www.linkedin.com/in/richardwallis
Twitter: @rjw

On 2 July 2018 at 22:42, Dan Brickley <notifications@github.com> wrote:

> LGTM. How should we handle updates to docs/releases.html ... we often end
> up with changes merged in that don't have corresponding releases text. But
> I think we've had conflicts when several parallel PRs all try to propose
> edits to the releases page. Maybe drafting the releases text here is a
> reasonable midway? /cc @vholland <https://github.com/vholland>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/schemaorg/schemaorg/pull/1979#issuecomment-401946782>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHVdII5srpsUPeU2Xv5yEwMU2sZ9ex7Pks5uCpO9gaJpZM4U8wUe>
> .
>

RichardWallis commented:
Moved proposed new types (Manuscript, Poster, Drawing, Play, SheetMusic) from 'bib' section into pending.  Updated text in releases.html to reflect this.
